### PR TITLE
Feat: Display output id in output

### PR DESCRIPTION
**Pull request by abbasabid**
# Description of change

- Display output id in outputs.
- Fix memory leak warning in associated output table.

Aims to fix https://github.com/iotaledger/explorer/issues/358

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## Change checklist

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by abbasabid** [comment]
This is how it looks now:

Screenshot 2022-06-27 at 4 41 40 PM

File: client/src/app/components/stardust/Output.tsx [hunk 1]
```diff
@@ -279,6 +286,24 @@ class Output extends Component<OutputProps, OutputState> {
         );
     }
 
+    /**
+     * Get output id.
+     * @param type The type to get the output id for.
+     * @returns output id.
+     */
+    private getOutputIdbyType(type: number) {
```
Comment (msarcev) [inline client/src/app/components/stardust/Output.tsx:294]: Please fix typo in function name here.

File: client/src/app/components/stardust/Output.tsx [hunk 2]
```diff
@@ -279,6 +286,24 @@ class Output extends Component<OutputProps, OutputState> {
...
+        switch (type) {
+            case ALIAS_OUTPUT_TYPE:
+                return this.buildAddressForAliasOrNft();
+            case FOUNDRY_OUTPUT_TYPE:
+                return this.buildFoundyId();
+            case NFT_OUTPUT_TYPE:
+                return this.buildAddressForAliasOrNft();
```
Comment (msarcev) [inline client/src/app/components/stardust/Output.tsx:301]: Case `ALIAS_OUTPUT_TYPE` and `NFT_OUTPUT_TYPE` can be collapsed in one case, as the call the same funcion.

File: client/src/app/components/stardust/Output.tsx [hunk 3]
```diff
@@ -279,6 +286,24 @@ class Output extends Component<OutputProps, OutputState> {
...
+            default:
+                return "Unknown Output Id";
```
Comment (msarcev) [inline client/src/app/components/stardust/Output.tsx:303]: What about Basic output type ?
image

File: client/src/app/components/stardust/Output.tsx (hunk 1 above)
Comment (abbasabid) [reply client/src/app/components/stardust/Output.tsx]: Resolved

File: client/src/app/components/stardust/Output.tsx (hunk 3 above)
Comment (abbasabid) [reply client/src/app/components/stardust/Output.tsx]: Removed that function and use output id from props.

**Comment by msarcev** [comment]
LGTM.
Needs links to be updated to point to "Output page" which we don't have atm. So it will be finished after https://github.com/iotaledger/explorer/issues/370.

**Review by msarcev — APPROVED** [review]

**Pull request merged**